### PR TITLE
Useful scripts for interacting with WPT logs

### PR DESCRIPTION
**Pull request by jdm**
These are scripts I used to generate the list in https://github.com/servo/servo/issues/24828, analyze test failures for #23290, and disable slow tests.

### CODE REVIEW

File: etc/wpt-summarize.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+full_search = len(sys.argv) > 3 and sys.argv[3] == '--full'
+
+with open(sys.argv[1]) as f:
+    data = f.readlines()
+    thread = None
+    for entry in data:
+        entry = json.loads(entry)
+        if thread and "thread" in entry:
+            if entry["thread"] == thread:
+                print(json.dumps(entry))
```
Comment (Manishearth) [inline etc/wpt-summarize.py]: won't this skip test_starts from other threads?

**Review by Manishearth — APPROVED** [review]
Could each script have a comment explaining what it does?

File: etc/wpt-summarize.py (hunk 1 above)
Comment (jdm) [reply etc/wpt-summarize.py]: I'm not sure I understand the concern here. The purpose of this script is to extract the output of a single test, so we don't care about other test_start events.

Comment (Manishearth) [reply etc/wpt-summarize.py]: oh! i think i hadn't fully understood that part

**Comment by jdm** [comment]
Added comments.

**Pull request merged**